src/Akka.Hosting/Logging/LoggerSetup.cs

```diff
@@ -0,0 +1,108 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="LoggerSetup.cs" company="Akka.NET Project">
+//      Copyright (C) 2013-2022 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using Akka.Configuration;
+using Akka.Dispatch;
+using Akka.Event;
+
+namespace Akka.Hosting.Logging
+{
+    public sealed class LoggerSetup
+    {
+        private readonly List<Type> _loggers = new List<Type> { typeof(DefaultLogger) };
+        internal AkkaConfigurationBuilder Builder { get; }
+
+        internal LoggerSetup(AkkaConfigurationBuilder builder)
```

Comment:
I thought this was, originally a `Setup` class from Akka.NET - now that I know it's not, fine having it take a dependency on `AkkaConfigurationBuilder`.